benchmarks/src/main/java/SimpleTrace.java

```diff
@@ -0,0 +1,56 @@
+package org.corfudb.benchmarks;
+
+import lombok.extern.slf4j.Slf4j;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+public class SimpleTrace {
+    String name; //name of the tracing object
+    long total; //sum of latencise for all operations in microseconds
+    long cnt;   //total number of the operations.
+    long start; //the current operation start time in nanoseconds
+
+    SimpleTrace(String name) {
+        this.name = name;
+        total = 0;
+        cnt = 0;
+        start = System.nanoTime();
+    }
+
+    void start() {
+        start = System.nanoTime();
+    }
+
+    void end() {
+        long interval = TimeUnit.NANOSECONDS.toMicros(System.nanoTime() - start);
+        //avoid to overflow the value, do a reset. Keep half of the history.
+        if (Long.MAX_VALUE - total < interval) {
+            this.log(false);
+            cnt = cnt/2;
+            total = total/2;
+        }
+
+        total += interval;
+        cnt++;
+    }
+
+    void log(boolean infoMode) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651469)